nucleus/common/glassfish-api/src/main/java/fish/payara/api/admin/config/NameGenerator.java

```diff
@@ -37,106 +37,53 @@
  * only if the new code is made subject to such option by the copyright
  * holder.
  */
-package fish.payara.nucleus.hazelcast;
 
-import java.util.Arrays;
-import java.util.HashMap;
-import java.util.List;
-import java.util.Map;
+package fish.payara.api.admin.config;
+
 import java.util.concurrent.ThreadLocalRandom;
 
-/**
- * Class to generate random names for Payara Micro instances.
- * <p>
- * There are over 14,000 different possible names.
- * @author Andrew Pielage
- */
-public class NameGenerator
-{
-    /** 161 different possible adjectives to describe a fish */
-    private final String[] adjectives = new String[]{"Adorable", "Adventurous", "Aggressive", "Alert", "Beautiful", 
-            "Blue-eyed", "Bloody", "Blushing", "Bright", "Clear", "Cloudy", "Colourful", "Crowded", "Cute", "Dark", 
-            "Drab", "Distinct", "Dull", "Elegant", "Excited", "Fancy", "Filthy", "Glamorous", "Gleaming", "Gorgeous", 
-            "Graceful", "Grotesque", "Handsome", "Heavy", "Homely", "Light", "Magnificent", "Misty", "Motionless", "Muddy", 
-            "Old-fashioned", "Plain", "Poised", "Precious", "Quaint", "Shiny", "Sparkling", "Spotless", "Stormy", 
+public class NameGenerator {
+    // 161 different possible adjectives to describe a fish
+    protected static final String[] adjectives = new String[]{"Adorable", "Adventurous", "Aggressive", "Alert", "Beautiful",
```

Comment:
I made all of these static since we don't really ever need an actual instance of this class (or its new subclasses)